### DISCUSSION TITLE
Why Native Queries enterprise with 2.31.0?

### DISCUSSION THREAD

**Discussion by fschuermeyer** [hasura/graphql-engine#9814] [Question] [answered]
Hello,

before Version 2.31.0 the Native Queries free for all. After the Update to 2.31.0 the Feature is enterprise only. In the Release Notes of 2.31.0 (https://github.com/hasura/graphql-engine/releases/tag/v2.31.0) I can't find any Information about it?

What are the Plans? I think the splitting between Enterprise and without is ok. But to make a feature later to Enterprise, I think it's a very bad practice.

Was this a mistake, or can you tell us something about it? In the Documentation stands Native Querys CE Version supported with 2.28.0 - i hope it's a mistake.

![image](https://github.com/hasura/graphql-engine/assets/25571323/5624f826-56e5-4ace-8ebc-d5d45f754bd4)
![image](https://github.com/hasura/graphql-engine/assets/25571323/322bd007-b3cd-4277-8e60-a919e181dbaa)

**Comment by SamirTalwar** [comment]
Heya. Native Queries is part of the free, open-source Hasura GraphQL Engine for PostgreSQL, and this should not have changed in v2.31.0.

Are you getting some kind of error message? Can you explain what's happening for you?

**Reply by fschuermeyer** [reply]
hay i updated my post and added some screenshots i see a message to enable Enterprise to get the feature. I have the same issue on different instances on different servers

**Reply by m4ttheweric** [reply]
Hey @fschuermeyer I am looking into this right now, I will update you with more information once we have an understanding of what's going on.

Can you share the URL you are accessing as well?

**Reply by m4ttheweric** [reply, marked as answer]
@fschuermeyer We have identified the issue and are working on deploying a hotfix ASAP. Thanks you for your time reporting this!

**Reply by fschuermeyer** [reply]
Thanks for the work on this! Sorry i see your Question to late.

**Reply by m4ttheweric** [reply]
@fschuermeyer FYI, due to it being so close to the weekend we are going to be deploying the fix on Monday. I will follow up with you when I know it's out!

**Reply by m4ttheweric** [reply]
@fschuermeyer So, we actually were able to get this pushed out just now. You should see the fix live within the hour I believe. 😁

**Reply by fschuermeyer** [reply]
thanks, for the very fast support. Incredible!
